src/app.ts

```diff
@@ -21,20 +20,18 @@ async function main() {
     const config: PluginConfig = defaultPluginConfig;
     populatePluginConfigFromEnv(config, env);
     if (!config.rendering.chromeBin && (process as any).pkg) {
-      //@ts-ignore
-      const executablePath = puppeteer.executablePath() as string;
-
-      if (executablePath.includes(chromeFolderPrefix)) {
-        const parts = executablePath.split(path.sep);
-        while (!parts[0].startsWith(chromeFolderPrefix)) {
-          parts.shift();
-        }
-
-        config.rendering.chromeBin = [path.dirname(process.execPath), ...parts].join(path.sep);
-      } else {
-        // local chrome installation in dev mode
-        config.rendering.chromeBin = env['PUPPETEER_EXECUTABLE_PATH'];
-      }
+      const execPath = path.dirname(process.execPath);
+      const chromeInfoFile = fs.readFileSync(path.resolve(execPath, 'chrome-info.json'), 'utf8');
+      const chromeInfo = JSON.parse(chromeInfoFile);
```

Comment:
We'd probably handle the possible errors here and maybe default to executable path from puppeteer (the one in the `.cache` folder)?